### DISCUSSION TITLE
esp32 of the same model as lora but without lora it doesn't work

### DISCUSSION THREAD

**Discussion by wesioplayconstructor** [darthcloud/BlueRetro#487] [Q&A]
I recorded the firmware correctly on the ESP32 but it doesn't respond, I put 3 different versions, and nothing, I put another project to test and it gave life, but the blue retro doesn't do anything, it doesn't appear on the cell phone, it doesn't connect the controller, I don't understand.

**Comment by darthcloud** [comment]
Get the log from ESP32 after you flashed the firmware and pressed EN to reboot the ESP32:

https://github.com/darthcloud/BlueRetro/wiki/Getting-BlueRetro-debug-logs-via-Serial-port-Windows-10

**Reply by wesioplayconstructor** [reply]
I resumed today, I did what you said, just a bunch of characters: ▒▒▒▒▒▒▒▒▒▒b▒▒▒▒2l B▒ۛ▒ld▒▒d▒▒▒$▒▒ې
▒▒▒l`b▒ېے▒$l▒▒▒▒▒ے▒ll▒▒▒"▒
▒▒▒l`c▒܃▒▒l$▒▒▒ld▒▒▒▒dl▒▒▒▒l
▒ ▒▒2d`b▒܂ے▒|▒Bb▒▒▒s▒$▒▒▒▒B▒
▒▒▒l b afraid▒ۚdlГll▒▒▒`~▒l▒▒Ӑ$▒▒▒c threatened▒▒▒d$▒▒▒$l▒▒▒d$▒▒▒▒▒▒▒b de▒ll ▒dl▒▒ ~▒ll▒▒▒b▒
▒▒▒l`b▒ې▒▒ےdd▒▒▒$d▒Ӓ▒d▒▒Ӑ

**Reply by wesioplayconstructor** [reply]
in another google chrome terminal it is detecting the crystal at 26mhz

**Reply by wesioplayconstructor** [reply]
find out my crystal is 26 mhz, WHAT DO I DO?

**Comment by darthcloud** [comment]
The only board supported is list d in the BOM here:
https://github.com/darthcloud/BlueRetro/wiki/BlueRetro-DIY-Build-Instructions#bill-of-materials

Buy that
